src/sentry/event_manager.py

```diff
@@ -1136,8 +1137,45 @@ def _save_aggregate(self, event, hashes, release, **kwargs):
                 event.event_id,
             )
 
+        self._handle_owner_assignment(project, group, event)
```

Comment:
just calling this out for folks to eyeball: this is in the processing path for aggregates so performance might be a question